doc/usage/markdown.rst

```diff
@@ -22,26 +22,32 @@ Configuration
 
 To configure your Sphinx project for Markdown support, proceed as follows:
 
-#. Install *recommonmark*::
+#. Install the Markdown parser *recommonmark* from its source on GitHub::
 
-      pip install recommonmark
+      pip install git+https://github.com/rtfd/recommonmark
```

Comment:
Unfortunately, recommonmark has not been released for a long time. So this example uses HEAD version.
But many users use released version. So we have to mention about it.

Hopefully, it would be nice if recommonmark team will ship a new release...